frame/staking/src/pallet/impls.rs

```diff
@@ -701,7 +727,11 @@ impl<T: Config> Pallet<T> {
 						.map_or(true, |spans| submitted_in >= spans.last_nonzero_slash())
 				});
 				if !targets.len().is_zero() {
-					all_voters.push((nominator.clone(), Self::weight_of(&nominator), targets));
+					all_voters.push((
```

Comment:
if you want to be pedantic, you should also make the return type of this function bounded since it will eventually be stored in `pallet-election-provider-multi-phase::Snapshot`, just maybe coordinate with @emostov since he might be working on the same.

Reply:
That would actually need quite a lot of changes, it's probably better to have it in another PR. Please let me know if there is anything else that is needed?